mon-pix/app/router.js

```diff
@@ -56,10 +56,10 @@ Router.map(function() {
     this.route('skill-review', { path: '/:campaign_code/resultats/:assessment_id' });
   });
 
-  this.route('competence-details', { path: '/competences/:scorecard_id' });
-  this.route('competences', function() {
-    this.route('resume', { path: '/:competence_id/evaluer' });
+  this.route('competences', { path: '/competences/:competence_id' }, function() {
+    this.route('details', { path: '/scorecard/:scorecard_id' });
```

Comment:
Je l'impression que cette route mélange deux notions : `competences` et `scorecard` (même si on utilise un scorecard_id pour le détail d'une compétence.

Je suggère plutôt `this.route('details', { path: '/details' });` ce qui donnerait au final une route du type `/competences/:competence_id/details`